wsd-frontend/src/components/wsd/Profile/ProfileTabsList/ProfileTabsList.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+import { useEffect, useState } from 'react'
+
+import * as Icons from 'lucide-react'
+
+import { Tabs, TabsList, TabsTrigger } from '@/components/shadcn/tabs'
+
+export function ProfileTabsList() {
+  const currentPath = usePathname()
+  const [tabs] = useState([
+    { name: 'Profile', href: '/profile/details', icon: Icons.User },
+    { name: 'Password', href: '/profile/password', icon: Icons.Lock },
+    { name: 'Emails', href: '/profile/emails', icon: Icons.Mail },
+    { name: 'Connections', href: '/profile/connections', icon: Icons.Link },
+  ])
+  const defaultValue = tabs[0].href
+
+  const [value, setValue] = useState(defaultValue)
+  useEffect(() => {
```

Comment:
If we already have the currentPath and an object of tab values can we not ditch both the useState and useEffect and just use the currentPath value to query the object and return the result? 

I don't know if the useState and useEffect does anything but add unnecessary code here. 

(I'm not saying that they are definitely unnecessary but I might be slightly uneducated on the whole react render cycle, I thought we didn't need states for constant objects like the one we have here and we can just use the value directly instead of doing it in useEffect since the entire component is going to be rerendered anyway when the path changes. ) 

(Also unrelated note: thank you for all the PRs, I'm a bit swamped with work stuff but I'll review all of these asap so we can merge these)